README.rst

```diff
@@ -289,26 +289,26 @@ To record an LTTng kernel trace suitable for the LTTng analyses:
 
       sudo lttng enable-event --kernel --channel=chan block_bio_backmerge
       sudo lttng enable-event --kernel --channel=chan block_bio_remap
-      sudo lttng enable-event --kernel --channel=chan block_dirty_buffer
       sudo lttng enable-event --kernel --channel=chan block_rq_complete
       sudo lttng enable-event --kernel --channel=chan block_rq_issue
       sudo lttng enable-event --kernel --channel=chan irq_handler_entry
       sudo lttng enable-event --kernel --channel=chan irq_handler_exit
+      sudo lttng enable-event --kernel --channel=chan irq_softirq_entry
+      sudo lttng enable-event --kernel --channel=chan irq_softirq_exit
+      sudo lttng enable-event --kernel --channel=chan irq_softirq_raise
+      sudo lttng enable-event --kernel --channel=chan kmem_mm_page_alloc
+      sudo lttng enable-event --kernel --channel=chan kmem_mm_page_free
       sudo lttng enable-event --kernel --channel=chan lttng_statedump_block_device
       sudo lttng enable-event --kernel --channel=chan lttng_statedump_file_descriptor
       sudo lttng enable-event --kernel --channel=chan lttng_statedump_process_state
-      sudo lttng enable-event --kernel --channel=chan mm_page_alloc
-      sudo lttng enable-event --kernel --channel=chan mm_page_free
```

Comment:
I would keep these 2 because they are the old name of the kmem_mm_page_alloc/free events, before the namespacing cleanup in lttng-modules.